app/models/item.js

```diff
@@ -390,6 +394,14 @@ export default cloudinaryUrl.extend(GradeMixin, {
     }
   ),
 
+  unreadMessagesCount: Ember.computed("messages.@each.state", function() {
+    return this.get("messages").filterBy("state", "unread").length;
+  }),
+
+  hasUnreadMessages: Ember.computed("unreadMessagesCount", function() {
```

Comment:
Can we use `gt` computed property here ?

Reply:
`.gt` not available in Ember 2.15